scripts/imageIDI.py

```diff
@@ -215,11 +208,11 @@ def main(args):
                 hdu.header['EPOCH'] = 2000.0 + (jdList[0] - 2451545.0) / 365.25
                 hdu.header['CTYPE1'] = 'RA---SIN'
                 hdu.header['CRPIX1'] = img.shape[0]//2+1
-                hdu.header['CDELT1'] = -1 * numpy.degrees(pixel_size)
+                hdu.header['CDELT1'] = -1 * np.degrees(pixel_size)
                 hdu.header['CRVAL1'] = pc_ra
                 hdu.header['CTYPE2'] = 'DEC--SIN'
                 hdu.header['CRPIX2'] = img.shape[1]//2+1
-                hdu.header['CDELT2'] = numpy.degrees(pixel_size)
+                hdu.header['CDELT2'] = np.degrees(pixel_size)
```

Comment:
Same thing here as with `lsl.imging.utils` with regards to how correct the WCS is.

Reply:
This and `lsl.imaging.utils` are at least self consistent now.